lib/src/observable.dart

```diff
@@ -22,7 +22,7 @@ abstract class Observable<C extends ChangeRecord> {
   final ChangeNotifier<C> _delegate = new ChangeNotifier<C>();
 
   // Whether Observable was not given a type.
-  final bool _isNotGeneric = C == dynamic;
+  final bool _supportsPropertyChanges = PropertyChangeRecord is C;
```

Comment:
The is looks backwards to me... Can you explain how This works? Current comment doesn't seem to match